batch/build-batch-worker-image-startup.sh

```diff
@@ -39,10 +39,12 @@ curl -fsSL "https://github.com/GoogleCloudPlatform/docker-credential-gcr/release
 
 # avoid "unable to get current user home directory: os/user lookup failed"
 export HOME=/root
-docker-credential-gcr configure-docker
 
-GCP_PROJECT=$(curl -s -H "Metadata-Flavor: Google" "http://metadata.google.internal/computeMetadata/v1/project/project-id")
-docker pull gcr.io/$GCP_PROJECT/ubuntu:18.04
+DOCKER_PREFIX=$(curl -s -H "Metadata-Flavor: Google" "http://metadata.google.internal/computeMetadata/v1/instance/attributes/docker_prefix")
+
+docker-credential-gcr configure-docker --include-artifact-registry
+
+docker pull ${DOCKER_PREFIX}/ubuntu:18.04
```

Comment:
Could this use `DOCKER_ROOT_IMAGE`?

Reply:
Actually `build-batch-worker-image-startup.sh` doesn't have access to DOCKER_PREFIX or DOCKER_ROOT_IMAGE. I switched to applying `jinja2_render.py` to insert them into the script.